src/egon/data/airflow/dags/pipeline_status_quo.py

```diff
@@ -495,7 +497,7 @@
             fill_etrago_generators,
             create_ocgt,
             gas_production_insert_data,
-            insert_data_ch4_storages,
+            #insert_data_ch4_storages,
```

Comment:
Why did you comment these lines?

Reply:
The task `insert_data_ch4_storages` fails every time we run Airflow and have to be marked as successful in order to continue with other tasks. It actually has no effect on the status2019 scenario since it is also executed for eGon2035 and eGon100RE. Reference [here](https://github.com/openego/powerd-data/blob/2676d99c26a5fb720ecbb1bdf418cb837e5970b6/src/egon/data/datasets/ch4_storages.py#L335-L336)

Reply:
Task `insert_data_ch4_storages` was deleted for the pipeline_status_quo d898f7ab942694966bf1686bdf6308ce0ecd75a5